docs/refarch/boundedContexts/reporting/index.md

```diff
@@ -1,21 +1,66 @@
-# {name} BC
+# Reporting BC
 
-{overview}
+The Reporting strategy for this Reference Architecture, is to describe the generic mechanisms by which client BCs' data can be persisted, and kept up to date, in a Reporting Data Store, so users and systems, can later consume this data directly from the Reporting Data Store, or through any Reporting and/or Dashboarding tools connected to the Reporting Data Store.  
+
+This Reporting Data Store must be write-only from the switch's perspective and read-only by external components.
+
+The data models on the Reporting Data Store can be different from the internal operational data models that the switch uses; where relevant, for performance or other reasons, multiple models of the same data can be made available in the Reporting Data Store - akin to multiple projections or views.
+A Client BC Reporting Component provided by the switch, will be translating internal events and internal data models to the external data store models - This component can be replaced or, there can even exist many of these for a single Client BC.
+Such component must exists in the Reporting BC for any Client BC's who's data is made available in the Reporting Data Store. 
+Any direct data sending, or fetching, from the source Client BC, or its internal data stores, to the Reporting Data Store, constitutes a violation of the decoupling principle and will negatively affect the maintainability of the system by virtue of its tight coupling.
```

Comment:
Can we clarify or expand on `Any direct data sending` please.

In the previous sentence, it indicates that we can do this with a Reporting component specific to each BC to send the data.

Perhaps by this, you mean that no Client BC should "write" directly to the Reporting BC's Data store?